AcadeMe/views.py

```diff
@@ -1,16 +1,28 @@
 from django.shortcuts import render
+from AcadeMeData.models import University, Degree
 
 
 def app_layout(request):
+    # context = {}
+    # if request.user.is_authenticated:
+    #     context['navbar_links'] = {f"Welcome {request.user.username}": "#", "Logout": "/logout"}
+    # else:
+    #     context['navbar_links'] = {"Login": "/login", "Register": "/register"}
```

Comment:
I think you do not want to Approve this PR :)
Please delete it

Reply:
uhhh I forgot that .... :expressionless:

Done.